packages/instrumentation-typeorm/package.json

```diff
@@ -32,31 +32,25 @@
         "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
     },
     "dependencies": {
-        "@opentelemetry/core": "^0.15.0",
-        "@opentelemetry/semantic-conventions": "^0.15.0",
-        "shimmer": "^1.2.1"
+        "@opentelemetry/instrumentation": "^0.15.0",
```

Comment:
I see you removed the dependency on `@opentelemetry/core`, which is found in other instrumentation libraries.
We should at least be dependent on `@opentelemetry/api` (probably an old issue)

Reply:
fixed